src/PanelTraits/Boxes.php

```diff
@@ -0,0 +1,133 @@
+<?php
+
+namespace Backpack\CRUD\PanelTraits;
+
+trait Boxes
+{
+    public $sideBoxesEnabled = false;
+    public $boxesOptions = [];
+
+    public function enableSideBoxes()
+    {
+        $this->sideBoxesEnabled = true;
+
+        return $this->sideBoxesEnabled;
+    }
+
+    public function disableSideBoxes()
```

Comment:
This sounds like it **does** something but I think it's just a setter.

Should getters and setters conform to the examples set in https://github.com/Laravel-Backpack/CRUD/blob/master/src/CrudPanel.php#L84 (i.e. have getXYZ and setXYZ)?